tests/features/support/steps.js

```diff
@@ -223,11 +225,59 @@ When(/I check the "([^"]+)" checkbox/, async (label_text) => {
   *    Very limited. Anything more is a future feature.
   * 
   * "checkbox": label that contains checkbox-like behavior.
+  * 
+  * May switch to using the below instead - almost same code, but
+  *    its text has to match exactly and it turns out clicking labels
+  *    works for more than one thing.
   */
   let checkbox = await scope.page.waitFor( `label[aria-label*="${ label_text }"]` );
   await checkbox.click();
 });
 
+When('I pick the {string} option', async (label_text) => {
+  /* Clicks the first label "containing" the "label text".
+  *    Very limited. Anything more is a future feature.
+  */
+  let choice = await scope.page.waitForSelector( `label[aria-label="${ label_text }"]` );
+  await choice.click();
+
+  await scope.waitForShowIf(scope);
+});
+
+// TODO: Should it be 'containing', or should it be exact? Might be better to be exact.
+// TODO: Should we have a test just for the state of MA to be selected? Much easier... Or states in general
+When('I select the {string} option from the {string} choices', async (choice_text, label_text) => {
+  /* Selects the option having exactly the text `choice_text`
+  * in the <select> with the label "containing" the `label text`.
+  *    Finding one out of a bunch is a future feature.
+  * 
+  * Note: `page.select()` is the only way to click on an element in a <select>
+  */
+  // Make sure ajax is finished getting the items in the <select>
+  await scope.page.waitForSelector('option');
+
+  // The <label> will have the `id` for the <select> we're looking for
+  let [label] = await scope.page.$x(`//label[text()="${label_text}"]`);
```

Comment:
Are you sure that this XPATH will return an array with only one element? I'm afraid this line might break if this could return more than one element. We could do something like `let [label, ...extras] = `

Reply:
In js, that syntax should just pick up the first element in any array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Description

Reply:
Ah TIL!